scripts/linters/general_purpose_linter.py

```diff
@@ -99,8 +99,8 @@
 BAD_PATTERNS_REGEXP = [
     {
         'regexp': re.compile(r'TODO[^\(]*[^\)][^:]*[^A-Z]+[^\w]*$'),
-        'message': 'Please assign TODO comments to a user '
-                   'in the format TODO(username): XXX. ',
+        'message': 'Please assign TODO comments to an issue '
+                   'in the format TODO(issuenum): XXX. ',
```

Comment:
Thanks for fixing this!

Not sure about the wording here i.e, `assign` ... `to an issue`, is that expected? Also, should we change `TODO(issuenum)` to `TODO(#issuenum)`?

Reply:
Thanks for the catch -- done! I changed "assign" to "link".